frontend/src/components/pages/VolunteerDashboard/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { Container } from "@chakra-ui/react";
+import React, { useContext, useEffect, useState } from "react";
+
+import VolunteerAPIClient from "../../../APIClients/VolunteerAPIClient";
+import AuthContext from "../../../contexts/AuthContext";
+import { Status } from "../../../types/AuthTypes";
+import PendingPage from "./PendingPage";
+import ScheduledVolunteerShiftsPage from "./ShiftsPage";
+
+const VolunteerDashboard = () => {
+  const [volunteerStatus, setVolunteerStatus] = useState<Status>();
+  const { authenticatedUser } = useContext(AuthContext);
```

Comment:
same thing over here, lets use AuthenticatedVolunteerContext and we will not need getVolunteerData() since we already have the status in that object